### PR TITLE
updated quaconfig

### PR DESCRIPTION
**Pull request by liorella-qm**
I added functions and some docstrings to quaconfig at the request of Yonatan.
We probably need to merge it with Satya's work, but for now let's just update

### CODE REVIEW

**Comment by galwiner** [comment]
Satya's work? meaning the qua config wrapper?

File: entropylab_qpudb/_quaconfig.py [hunk 1]
```diff
@@ -133,18 +142,52 @@ def update_intermediate_frequency(
...
+    def set_output_dc_offset_by_element(self, element: str, port: str, offset: float) -> None:
+        """
+        Set a DC offset value by element
+
+        :param element: Name of the element
+        :param port: Name of the port. Can be either 'single' if element has `singleInput` or 'I', 'Q' if element has mixed inputs.
+        :param offset: offset value to set
+        """
+        port = self.get_port_by_element(element, port)
+        self.data["controllers"][port[0]]["analog_outputs"][port[1]]["offset"] = offset
```
Comment (galwiner) [inline entropylab_qpudb/_quaconfig.py:193]: is this correct? seems weird that port is under controllers and also under analog input

Comment (liorella-qm) [reply entropylab_qpudb/_quaconfig.py:193]: you mean that I use the term "port" for both the input to the function (i.e. the input of the element) and for the actual port? You're right, it's weird.
I think the mapping is "get controller port from element input" - this is what makes sense. I'll change it and you can review again

**Review by qguyk — CHANGES_REQUESTED** [review]
@liorella-qm 
maybe add some tests or remove the empty tests file

**Comment by liorella-qm** [comment]
@qguyk added some tests - still many more needed
@galwiner please see changes

**Review by qguyk — APPROVED** [review]

**Review by galwiner — APPROVED** [review]
